src/widgets/archivelistwidget.cpp

```diff
@@ -4,6 +4,7 @@
 #include "ui_archiveitemwidget.h"
 
 #include <QMessageBox>
+#include <QKeyEvent>
```

Comment:
The Title of the PR says Textlabel fixes. Can you break unrelated changes from now on into separate PRs?